client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -106,7 +106,7 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 					<ControlError error={ address1Error } />
 
 					<TextControl
-						label={ __( 'Address line 2' ) }
+						label={ __( 'Address line 2 (optional)' ) }
```

Comment:
There is a `help` prop on `TextControl` but it positions the text element outside of and after the field group. We could move it with a CSS hack but this seemed more accessible, and closer to the design.

Reply:
yeah I think this is fine